components/encoder/AMS/ams_as5048_test.go

```diff
@@ -1,12 +1,27 @@
 package ams
 
 import (
+	"context"
 	"math"
 	"testing"
 
+	"github.com/edaniels/golog"
 	"go.viam.com/test"
+	"go.viam.com/utils"
+	"go.viam.com/utils/testutils"
+
+	"go.viam.com/rdk/components/board"
+	"go.viam.com/rdk/components/encoder"
+	"go.viam.com/rdk/config"
+	"go.viam.com/rdk/registry"
+	"go.viam.com/rdk/resource"
+	"go.viam.com/rdk/testutils/inject"
 )
 
+var model = resource.NewDefaultModel("AMS-AS5048")
+
+var writeData = make(map[byte]byte)
+
```

Comment:
[minor] rather than defining this at the file level, this should be initialized in the test case and passed as an argument to `setupDependenciesWithWrite`. Defining mutable structures outside of functions is in general an anti-pattern (with some exceptions)